docs/source/guides/tools/rpakpilotmodding.rst

```diff
@@ -0,0 +1,245 @@
+.. _importpilotmodel:
+
+Rpak Pilot Model Importing
+=====
+
+Tools:
+^^^^^
+
+Download each tool. 
+
+.. _Blender: https://www.blender.org/
+
+.. _Blender Source Tools: http://steamreview.org/BlenderSourceTools/
+
+.. _Titanfall VPK Tools: https://retryy.gitbook.io/tf2/Wiki/Tools/vpk-tools
+
+.. _Crowbar: https://steamcommunity.com/groups/CrowbarTool
+
+.. _Legion+: https://github.com/r-ex/LegionPlus
+
+.. _RePak: https://github.com/r-ex/RePak
+
+.. _mdlshit: https://github.com/headassbtw/mdlshit-binaries
+
+.. _Text Editors: https://retryy.gitbook.io/tf2/Wiki/Tools/general-pc-tools#text-editing
+
+
+
+- `Text Editors`_ - Just a text editor
+
+- `Titanfall VPK Tools`_ - VPK viewer, extractor, and repacker.
+            "Titanfall VPK Tool" is the vpk tool used. Made by cra0.
+  
+- `Crowbar`_ - Decompile, recompile, and a ton of other model and source asset stuff
+  
+- `Blender`_ - 2D/3D Workspace
+  
+- `Blender Source Tools`_ - Source Tools plugin for Blender
+  
+- `Legion+`_  - Texture, audio, shaderset, and material exporter
+               "Normal" Legion doesn't work
+  
+- `RePak`_ - Create Apex and Titanfall 2 .rpak for textures
+  
+- `mdlshit`_ - Magically convert .mdl to "titanfall 2" .mdl 
+
+Exporting the pilot model from the vpk
+^^^^^
+
+Locate your Titanfall 2 game folder
+-----
+
+For Origin you should find it in this directory ``Origin Games\Titanfall2``.
+
+For Steam you should find it in this directory ``Steam\steamapps\common\Titanfall2``.
+
+Open Titanfall VPK Tool
+-----
+
+At the top left press the ``Open`` button.
+Open the vpk folder then open ``englishclient_mp_common.bsp.pak000_dir.vpk``,
+Expand the ``models`` folder, then the ``humans`` folder, then click on the ``pilots`` folder.
+
+The files have different names that correspond to their pilot classes. Below is a clarifying table to help.
+
+====================    ===== 
+VPK .mdl name           Name
+====================    =====
+pilot_light_jester      Stim
+pilot_light_ged         Phase Shift
+pilot_medium_stalker    Holo Pilot
+pilot_medium_reaper     Pulse Blade
+pilot_medium_geist      Grapple
+pilot_heavy_roog        A-Wall
+pilot_heavy_drex        Cloak
+====================    =====
```

Comment:
This should probably also specify that _f models are the female ones and _m models are the male ones, for completeness and to avoid any potential confusion

Reply:
Will add this!